lib/acronym_cases.rb

```diff
@@ -0,0 +1,19 @@
+class AcronymCase < OpenStruct
+  def name
+    'test_%s' % expected.downcase
```

Comment:
`description` in the `canonical-data.json` file provides information about what the test is testing for.
Either: use the description as the test name.
add the description as a comment in the test cases.